dateparser/custom_language_detection/fasttext.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+import fasttext
+
+from dateparser_cli.fasttext_manager import fasttext_downloader
+from dateparser_cli.utils import dateparser_model_home, create_data_model_home
+
+
+_supported_models = ["large.bin", "small.bin"]
+_DEFAULT_MODEL = "small.bin"
+
+
+class _FastTextCache:
+    model = None
+
+
+def _load_fasttext_model():
+    if _FastTextCache.model:
+        return _FastTextCache.model
+    create_data_model_home()
+    model_path = None
+    downloaded_models = os.listdir(dateparser_model_home)
+    for downloaded_model in downloaded_models:
+        if downloaded_model in _supported_models:
+            model_path = os.path.join(dateparser_model_home, downloaded_model)
+    if not model_path:
+        fasttext_downloader("small")
+        return _load_fasttext_model()
+    if not os.path.isfile(model_path):
+        raise Exception('Fasttext model file not found')
+    _FastTextCache.model = fasttext.load_model(model_path)
+    return _FastTextCache.model
+
+
+def detect_languages(text, confidence_threshold):
+    _language_parser = _load_fasttext_model()
+    text = text.replace('\n', ' ').replace('\r', '')
+    language_codes = []
+    parser_data = _language_parser.predict(text)
+    for idx, langauge_candidate in enumerate(parser_data[1]):
+        if langauge_candidate > confidence_threshold:
+            language_code = parser_data[0][idx].replace("__label__", "")
+            language_codes.append(language_code)
```

Comment:
```suggestion
    for idx, language_candidate in enumerate(parser_data[1]):
        if language_candidate > confidence_threshold:
            language_code = parser_data[0][idx].replace("__label__", "")
            language_codes.append(language_code)
```

But I think this variable is not representing a `language_candidate` but a confidence probabilty.

Maybe it would be better to rename this to: `language_probability` or something similar.

Reply:
Done 👍🏻. I have changed it to `language_probability`